extensions/amp-story-dev-tools/0.1/amp-story-dev-tools-tab-logs.css

```diff
@@ -115,3 +115,32 @@ amp-story-dev-tools-tab-logs {
+  width: 260px !important;
+  height: 300px !important;
+  margin-bottom: 20px !important;
+}
+
+.i-amphtml-story-dev-tools-logs-success {
+  display: flex !important;
+  align-items: center !important;
+  justify-content: center !important;
+  flex-direction: column;
+  height: 100% !important;
+  white-space: pre !important;
```

Comment:
Might not need this.

Reply:
True, this was needed for adding the `\n` on the textContent before, but using templates we can do `<br>`